apps/Caskaydia Cove NF/install

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+cd /tmp
+mkdir cascadia
+cd cascadia
+wget https://github.com/ryanoasis/nerd-fonts/releases/download/v3.0.1/CascadiaCode.zip || error "failed to download font"
+unzip CascadiaCode.zip || error "failed to extract zip"
+sudo mkdir /usr/local/share/fonts
+sudo mkdir /usr/local/share/fonts/Caskaydia
+rm -- *Propo*
```

Comment:
I still think all fonts should be left. no reason to remove any since they just take up a bit more space.